docs/_docs/upgrading/0-to-2.md

```diff
@@ -1,140 +0,0 @@
----
-layout: docs
-title: Upgrading from 0.x to 2.x
-permalink: /docs/upgrading/0-to-2/
----
-
-Upgrading from an older version of Jekyll? A few things have changed in 1.0
```

Comment:
I worry that somebody might inherit a Jekyll site that hasn't been updated for years. Even if we aren't supporting 2.0, we should still provide a path for Jekyll 1.0 users to upgrade their sites to the latest and greatest.